Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -27,7 +27,6 @@ Calendar::Calendar(Core::DateTime date_time, Mode mode)
     : m_selected_date(date_time)
     , m_mode(mode)
 {
-    Vector<String> const weekday_names = { "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday" };
```

Comment:
This change should be squashed into the commit `Calendar: Add a Calendar settings dialog for the first day of the week`